app.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import 'highcharts';
+import ReactHighcharts from 'react-highcharts';
+
+/**
+ * Component representing a single char
```

Comment:
You probably meant chart instead of char.

Reply:
no, this component represents a 1 byte char

Reply:
jk fixed